packages/nx/src/command-line/release/changelog.ts

```diff
@@ -1116,7 +1116,7 @@ async function generateChangelogForWorkspace({
     changes,
     changelogEntryVersion: releaseVersion.rawVersion,
     project: null,
-    isVersionPlans: false,
+    isVersionPlans: !!nxReleaseConfig.versionPlans,
```

Comment:
Again I don't have a good explanation for why this was previously hardcoded to `false`... it should respect the config